Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -512,9 +512,10 @@ final class ChatViewController: SLKTextViewController {
 
     func registerTypingEvent(_ subscription: Subscription) {
         typingIndicatorView?.interval = 0
+        guard let user = AuthManager.currentUser() else { return Log.debug("Could not register TypingEvent") }
 
         SubscriptionManager.subscribeTypingEvent(subscription) { [weak self] username, flag in
-            guard let username = username else { return }
+            guard let username = username, username != user.username else { return }
```

Comment:
@rafaelks these changes got reverted by #1048 so not my fault :(

Reply:
Ouch! Sorry about that! Let's include it again. 👍